### ISSUE TITLE
"Today" button in tasks screen should not add GET params to url

### ISSUE THREAD

**Issue by jaragunde** [Igalia/phpreport#411]
The "today" button from the left calendar in tasks screen should redirect to `tasks.php` without any GET parameters to the url. It currently redirects to `tasks.php?date=2018-06-08`.

This is an interesting behavior to have when you reuse the same phpreport tab over and over during several days.

**Comment by jaragunde** [comment]
Reviewed as part of pull request #420:
```
commit b6b4a25fd70bc10ae515f7f4f36164215ed2f8e3
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Fri Aug 24 19:57:32 2018 +0200

    [#411] Load tasks page with no GET params when "today" is selected.

```

**Comment by jaragunde** [comment]
Reopening due to a regression. If we compare only the day of the month, it will send us to "today" if we try to open the same day on a different month. E.g. Today is Feb. 25th, if I try to open Jan. 25th it will send me back to February.

**Comment by jaragunde** [comment]
```
commit 796a5acc8c60faf4b4745048ee78324c41b5010a
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Mon Feb 25 12:07:06 2019 +0100

    [#411] Compare the entire date and not only the day of the month.
```

**Comment by jaragunde** [comment]
Landed new related commit, reviewed in PR #500:
```
commit eb1abf7516eda062d25d57524b29f899f333c936
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Tue Jun 15 11:26:45 2021 +0200

    [#411] Prevent redundant GET params when using prev/next date hotkeys.
